dbschema/prompt-variant-response.esdl

```diff
@@ -0,0 +1,34 @@
+module default {
+  type PromptVariantResponse extending Resource {
+    creator: User;
+    multi link responses := VariantResponse;
+  }
+
+  abstract type VariantResponse extending Resource {
+    variant: VariantResponse::Variant;
+    response: RichText;
+    creator: User;
+  }
+  
+  type PromptResponse extending Resource {
```

Comment:
This exists in app code, but has yet to have a use case. I'd just remove for now.
This is an alternate abstraction that has a single response instead of a response for each variant.